extractors/python_functions/iban_extraction/code_snippet.md

```diff
@@ -1,11 +1,10 @@
 ```python
 import re
-from typing import List, Tuple
 
 YOUR_ATTRIBUTE: str = "text" # only text attributes
 YOUR_LABEL: str = "iban"
 
-def iban_ext(record) -> List[Tuple[str, int, int]]:
+def iban_ext(record):
```

Comment:
why `ext` instead of `extractor` like with the other function names?

Reply:
Changed it!